docs/reference-manual/native-image/assets/build-output-schema-v1.json

```diff
@@ -0,0 +1,586 @@
+                    "examples": [
+                        "17.0.4+5-jvmci-22.2-b03"
+                    ]
+                },
+                "c_compiler": {
+                    "type": "string",
+                    "default": "",
+                    "title": "The C compiler used for native compilation and linkage",
+                    "examples": [
+                        "gcc (linux, x86_64, 9.3.0)"
+                    ]
+                },
+                "garbage_collector": {
+                    "type": "string",
+                    "default": "",
+                    "title": "The garbage collection algorithm in use for the generated native image",
+                    "examples": [
+                        "Serial GC"
+                    ]
+                }
+            },
+            "examples": [{
+                "name": "helloworld",
+                "graalvm_version": "GraalVM 22.2.0-dev Java 17 CE",
+                "java_version": "17.0.4+5-jvmci-22.2-b03",
+                "c_compiler": "gcc (linux, x86_64, 9.3.0)",
+                "garbage_collector": "Serial GC"
+            }]
+        },
+        "analysis_results": {
+            "type": "object",
+            "default": {},
+            "title": "Schema for the image generator statistics of the analysis phase",
+            "required": [
+                "classes",
+                "fields",
+                "methods"
+            ],
+            "properties": {
+                "classes": {
+                    "type": "object",
+                    "default": {},
+                    "title": "Number of classes detected during analysis phase",
+                    "required": [
+                        "total",
+                        "reachable",
+                        "reflection",
+                        "jni"
+                    ],
+                    "properties": {
+                        "total": {
+                            "type": "integer",
+                            "default": 0,
+                            "title": "The total number of classes",
+                            "examples": [
+                                3850
+                            ]
+                        },
+                        "reachable": {
+                            "type": "integer",
+                            "default": 0,
+                            "title": "The reachable reachable classes",
+                            "examples": [
+                                2839
+                            ]
+                        },
+                        "reflection": {
+                            "type": "integer",
+                            "default": 0,
+                            "title": "The classes registered for reflective access",
+                            "examples": [
+                                28
+                            ]
+                        },
+                        "jni": {
```

Comment:
`jni` will always be present (no longer conditional). In case there are `0` JNI classes, the method and field values will be `-1` to indicate their absence. This is now documented in the schema.

Reply:
I'm for reporting `0` when there are `0` JNI classes, Turning `0` into `-1` seems weird. But I'm happy to adjust that.